iidm/iidm-tck/src/test/java/com/powsybl/iidm/network/tck/AbstractVoltageLevelTest.java

```diff
@@ -60,7 +60,7 @@ public void baseTests() {
         assertEquals(500.0, voltageLevel.getNominalV(), 0.0);
 
         // Remove
-        voltageLevel.remove();
+        voltageLevel.remove();    // <---- TODO HERE
         assertNotNull(voltageLevel);
         try {
             voltageLevel.getSubstation();
```

Comment:
Here we have "Object has been removed in current variant" message. Our code is more generic. We don't precise which resource we can access to.